src/libs/ReportScrollManager/index.js

```diff
@@ -15,8 +16,15 @@ function scrollToIndex(index, isEditing) {
     if (isEditing) {
         return;
     }
-
-    flatListRef.current.scrollToIndex(index);
+    const sizeMap = flatListRef.current.props.sizeMap;
+    const clientHeight = flatListRef.current.getScrollableNode().clientHeight;
+    const paddingTop = parseFloat(flatListRef.current.getScrollableNode().childNodes[0].style.paddingTop || '0px');
+    const sizeArray = _.map(_.values(sizeMap), s => s.length);
+    const sumNum = _.reduce(sizeArray.slice(0, (index.index)), (acc, val) => acc + val);
+    flatListRef.current.scrollToOffset({
+        animated: index.animated,
+        offset: (sumNum + paddingTop + (index.viewOffset || 0)) - (clientHeight * (index.viewPosition || 0)),
```

Comment:
I think we should have offset as a separate variable for the ease of reading and debugging
```js
comst offset = sumNum + paddingTop + (index.viewOffset || 0)) - (clientHeight * (index.viewPosition || 0);
flatListRef.current.scrollToOffset({animated: index.animated, offset});
```